torch/utils/cpp_extension.py

```diff
@@ -153,6 +153,15 @@ def _join_rocm_home(*paths) -> str:
 
                               !! WARNING !!
 '''
+CUDA_MISMATCH_MESSAGE = '''
+The detected CUDA version ({0}) mismatches the version that was used to compile
+PyTorch ({1}). Please make sure to use the same CUDA versions.
+'''
+CUDA_MISMATCH_WARN = "The detected CUDA version ({0}) mismatches the minor version that was used to compile PyTorch ({1}). Most likely this shouldn't be a problem."
```

Comment:
```suggestion
CUDA_MISMATCH_WARN = "The detected CUDA version ({0}) has a minor version mismatch with the version that was used to compile PyTorch ({1}). Most likely this shouldn't be a problem."
```
the original version was a bit hard to parse, here is a better wording.